docs/Install.md

```diff
@@ -77,27 +77,39 @@ If you don't have an object storage provider, we recommend [MinIO](https://min.i
 
 ### Deploying Across Multiple Clusters
 
-By default, this tool will pull in data from the cluster in which it is running. The tool also supports collecting data across mulitple OpenShift clusters. In order to do this, we need to point the Pelorus instance at these other clusters.
+By default, this tool will pull in data from the cluster in which it is running. The tool also supports collecting data across mulitple OpenShift clusters. In order to do this, the thanos sidecar can be configured to read from a shared S3 bucket accross clusters. See [Pelorus Multi-Cluster Architecture](/page/Architecture.md) for details. You define exporters for the desired meterics in each of the clusters which metrics will be evaluated.  The main cluster's Grafana dashboard will display a combined view of the metrics collected in the shared S3 bucket via thanos.
 
-To do this, create a new variables file , `extra_prometheus_hosts.yaml`.  It is a yaml file with an array of entries with the following parameters:
+#### Configure main Pelorus instance to save and pull data from Thanos using a shared S3 Bucket.     
 
-* id - a description of the prometheus host (this will be used as a label to select metrics in the federated instance).
-* hostname - the fully qualified domain name or ip address of the host with the extra prometheus instance
-* password - the password used for the 'internal' basic auth account (this is provided by the k8s metrics prometheus instances in a secret).
+Sample primary values using AWS S3:
 
-For example:
+```
+bucket_access_point: s3.us-east-2.amazonaws.com
+bucket_access_key: <your access key>
+bucket_secret_access_key: <your secret access key>
+thanos_bucket_name: <bucket name here>```
+
+exporters:
+  instances:
+  ... 
+```
 
-    extra_prometheus_hosts:
-      - id: "ci-1"
-        hostname: "prometheus-k8s-openshift-monitoring.apps.ci-1.example.com"
-        password: "<redacted>"
+#### Configure exporters in additinal clusters to push data into the shared S3 Bucket.
```

Comment:
This title is misleading. It's not the exporters themselves that push data into object storage. Its the prometheus stack. The thing we want to show here is the values file for the second cluster, showing that the same bucket is being used, but different set of exporters deployed here vs in Dev.

Reply:
Here we are implying the production cluster, so the deployment exporter should be the only exporter deployed here.